test/cli-rc.test.js

```diff
@@ -22,7 +22,7 @@ test('cli', (t) => {
     const configFile = path.join(tmpDir, 'pino-pretty.config.js')
     fs.writeFileSync(configFile, 'module.exports = { translateTime: true }')
     const env = { TERM: 'dumb' }
-    const child = spawn(process.argv0, [bin], { env, cwd: tmpDir })
+    const child = spawn(process.argv[0], [bin], { env, cwd: tmpDir })
```

Comment:
Why these changes? `argv0` has been present since 6 -- https://nodejs.org/dist/latest-v12.x/docs/api/process.html#process_process_argv0

Reply:
Because `node test/cli-rc.test.js` was failing on my machine (Mac) on Node 12 if run without `tap`. I'm not sure why and I did not investigate.